test/integration/image-component/default/pages/invalid-placeholder-blur.js

```diff
@@ -4,7 +4,13 @@ import Image from 'next/image'
 const Page = () => {
   return (
     <div>
-      <Image id="invalid-placeholder-blur" src="/test.png" placeholder="blur" />
+      <Image 
+        id="invalid-placeholder-blur"
+        src="/test.png"
+        placeholder="blur"
+        width={200}
+        height={200}
```

Comment:
@jthcast Same here. This test should not be modified because you don't need width/height on a static import